app/assets/src/components/visualizations/Histogram.js

```diff
@@ -173,6 +176,57 @@ export default class Histogram {
     return this.options.barOpacity || 0.8;
   };
 
+  // Find the bar midpoint that is closest to hoverX, within hoverBuffer.
```

Comment:
Can you clarify what a "bar midpoint" is ?
This is not something that applies to all histograms, right?

Reply:
Changed to "bar x-center". It's the center of the bar on the x-dimension.